modules/logging_configs/main.tf

```diff
@@ -6,7 +6,7 @@ locals {
   fire_lens_container = <<EOF
   {
     "essential": true,
-    "image": "906394416424.dkr.ecr.${var.region}.amazonaws.com/aws-for-fluent-bit:stable",
+    "image": "514496004130.dkr.ecr.${var.region}.amazonaws.com/fluentbit-custom-eng:1.0.0",
```

Comment:
What about permissions for these services to pull that container? Don't we normally have to allow that on both sides?

Reply:
The task execution role for nerves_hub has access to pull all ECR resources.  As for the other side, I believe we are fine since we allow the eng account to pull ? 

https://github.com/smartrent/aws-fluent-bit-multiline-docker/blob/main/terraform/main.tf#L55

https://github.com/smartrent/aws-fluent-bit-multiline-docker/blob/main/terraform/main.tf#L18

https://github.com/smartrent/terraform-aws-account-ids/blob/main/outputs.tf#L1

@rraub Am I understanding this correctly?  I believe all permissions should be in place?  But I am looking into figuring out how to spin up a dev instance to test currently